schemas/bioscan-pool-xp-tube-to-traction/1-bioscan-pool-xp-tube-to-traction.avsc

```diff
@@ -0,0 +1,126 @@
+{
+    "namespace": "uk.ac.sanger.psd",
+    "type": "record",
+    "name": "ExportPoolXPTubeToTraction",
+    "doc": "A message to pass all the metadata required to export a Bioscan Pool XP tube to Traction.",
+    "fields": [
+        {
+            "name": "messageUuid",
+            "doc": "The UTF-8 byte encoded unique message ID.",
+            "type": {
+                "name": "version4MessageUuid",
+                "type": "fixed",
+                "size": 36
+            }
+        },
+        {
+            "name": "messageCreateDateUtc",
+            "doc": "Date (UTC) that the message was created. Represented as milliseconds since the Unix epoch.",
+            "type": {
+                "type": "long",
+                "logicalType": "timestamp-millis"
+            }
+        },
+        {
+            "name": "tubeBarcode",
+            "type": "string",
+            "doc": "The barcode for the tube."
```

Comment:
Should this be the human readable barcode or the machine barcode?

Reply:
Fair question and I don't know at this point.  Traction takes a barcode for the container you are submitting and it's not clear to me what the relevance is of that: https://github.com/sanger/tol-lab-share/pull/113/files#diff-e1dc92b9846ef1cf0ff96ba3f7f8313dcbe6f4dc4a90784a64b0436cda43fca1R18-R19

Steve previously told me: "We will definitely need tube barcode which will be the XP barcode."